tests/core/resource/test_preconditions.py

```diff
@@ -200,13 +200,55 @@ def test_post_if_none_match_star_succeeds_if_record_does_not_exist(self):
                 'field': 'new'}}
         self.resource.collection_post()  # not raising.
 
-    def test_post_if_match_star_succeeds_if_record_does_exist(self):
```

Comment:
should we have the tests on collection get/delete that always succeed with `If-Match:*` or `If-None-Match: *` ?